pandas/tests/io/excel/test_readers.py

```diff
@@ -994,15 +1031,20 @@ def test_reader_seconds(self, request, engine, read_ext):
         actual = pd.read_excel("times_1904" + read_ext, sheet_name="Sheet1")
         tm.assert_frame_equal(actual, expected)
 
-    def test_read_excel_multiindex(self, request, read_ext):
+    def test_read_excel_multiindex(self, request, engine, read_ext):
         # see gh-4679
-        if read_ext == ".xlsb":
+        if engine == "pyxlsb":
             request.node.add_marker(
                 pytest.mark.xfail(
                     reason="Sheets containing datetimes not supported by pyxlsb"
                 )
             )
 
+        if engine == "calamine" and read_ext == ".ods":
+            request.node.add_marker(
+                pytest.mark.xfail(reason="Last test fails in calamine")
+            )
```

Comment:
Sounds like this is a bug in calamine. Can you link to (& report if necessary) an issue.

Reply:
* https://github.com/tafia/calamine/issues/354